versions.yaml

```diff
@@ -35,6 +35,16 @@ docker_images:
     url: "https://hub.docker.com/_/nginx/"
     version: "1.15-alpine"
 
+  registry:
+    description: "Registry component for x86_64 and ARM"
+    url: "https://hub.docker.com/_/registry"
+    version: "2"
+
+  registry_ibm:
+    description: "Registry component for s390x and ppc64le"
+    url: "https://hub.docker.com/r/ibmcom/registry"
```

Comment:
is `https://` really needed? if `integration/kubernetes/init.sh `  is the only consumer I think we can remove it, wdyt?

Reply:
Yes, makes sense. I added it considering the format of how the images were being added in `versions.yaml`.
Probably we can change it to `name: "ibmcom/registry"` ?

Reply:
or maybe add a new one: `registry_url: "ibmcom/registry"`

Reply:
@devimc , updated it. PTAL

Reply:
thank you!